src/ngx_http_lua_timer.c

```diff
@@ -107,6 +114,20 @@ ngx_http_lua_ngx_timer_pending_count(lua_State *L)
 static int
 ngx_http_lua_ngx_timer_at(lua_State *L)
 {
+    return ngx_http_lua_ngx_timer_helper(L, 0);
+}
+
+
+static int
+ngx_http_lua_ngx_timer_every(lua_State *L)
+{
+    return ngx_http_lua_ngx_timer_helper(L, 1);
```

Comment:
Better add a code comment here for the TODO of returning a timer handle instead which can be passed to `ngx.timer`'s `cancel()` method to cancel the timer.

And yeah, in the near future, `ngx.timer.at()` should do the same. But cancelling a recurring timer is more common in practice.